src/schema/artist/shows.ts

```diff
@@ -84,26 +84,31 @@ export const ShowsConnectionField = {
         sort: "-end_at",
         total_count: true,
       })
-    ).then(({ body, headers }) => {
-      const totalCount = headers["x-total-count"]
-      const totalPages = Math.ceil(totalCount / size)
+    )
+      .then(({ body, headers }) => {
+        const whitelistedShows = showsWithBLacklistedPartnersRemoved(body)
+        return { body: whitelistedShows, headers }
```

Comment:
(Compare to L71, above.)

This may introduce some glitchy behavior around pagination on the (presumably small number of) artist pages that are affected by this.